### PR TITLE
refactor: Clean up styles

### PR DESCRIPTION
**Pull request by kazo0**
## PR Type

What kind of change does this PR introduce?

- Refactoring (no functional changes, no api changes)

## PR Checklist 
Please check if your PR fulfills the following requirements:

- Commits must be following the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- [ ] Tested UWP
- [ ] Tested iOS
- [ ] Tested Android
- [ ] Tested WASM
- [ ] Tested MacOS
- [ ] Contains **No** breaking changes
  > If the pull request contains breaking changes, commit message must contain a detailed description of the action to take for the consumer of this library. As explained by the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)

### CODE REVIEW

File: src/library/Uno.Cupertino/Styles/Controls/Slider.xaml [hunk 1]
```diff
@@ -21,7 +21,7 @@
 	<StaticResource x:Key="CupertinoSliderInlineTickBarFill"
 					ResourceKey="CupertinoBlueColor" />
 
-	<!-- Cupertino Slider Thumb Style -->
+	<!--  Cupertino Slider Thumb Style  -->
```
Comment (Xiaoy312) [inline src/library/Uno.Cupertino/Styles/Controls/Slider.xaml:24]: unnecessary change, you can change that your setting: Xaml Styler > Misc > Comment Padding > 1 (from 2)

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs [hunk 2]
```diff
@@ -56,8 +56,10 @@ public App()
 		/// <param name="e">Details about the launch request and process.</param>
 		protected override void OnLaunched(LaunchActivatedEventArgs e)
 		{
-			Uno.Material.Resources.Init(this, new ResourceDictionary() { Source = new Uri("ms-appx:///ColorPaletteOverride.xaml") });
```
Comment (Xiaoy312) [inline src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: Is there any particular reason as to why this line is expanded?

Comment (kazo0) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: Just made it match what we are doing in Gallery, for some reason otherwise the colors weren't being properly overridden

Comment (Xiaoy312) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: Can you try to restore the old `Init` method, and replace the 2nd argument by a func?:
```cs
public static void Init(Application app, Func<ResourceDictionary> colorPaletteOverride = null)
{
//...
if (colorPaletteOverride?.Invoke() is ResourceDictionary override)
{
     app.Resources.MergedDictionaries.Add(override);
}
//...
}
```
And, see if this works? My suspicion is that the order at which the res-dicts are initialized played a role here.

Comment (kazo0) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: Updated and seems to work

File: src/library/Uno.Cupertino/Styles/Controls/TextBlock.xaml [hunk 3]
```diff
@@ -1,148 +1,150 @@
...
+	<Style x:Key="CupertinoLargeTitle"
 		   TargetType="TextBlock"
 		   BasedOn="{StaticResource CupertinoBaseTextBlockStyle}">
 
 		<Setter Property="FontSize"
-				Value="45" />
+				Value="34" />
 		<Setter Property="LineHeight"
-				Value="55" />
+				Value="41" />
```
Comment (Guidemarcus) [inline src/library/Uno.Cupertino/Styles/Controls/TextBlock.xaml:35]: Those styles are not as designed on Figma. Are you sure about these changes?

Comment (kazo0) [reply src/library/Uno.Cupertino/Styles/Controls/TextBlock.xaml:35]: These are the numbers coming from the Human Interface Design Guidelines, I trust those more than the Figma file.

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs (hunk 2 above)
Comment (Xiaoy312) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: @MatFillion The mystery was finally solved!

Comment (MatFillion) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: Hmm interesting! Good find! You can look maybe we have an open issue we could close🚀🚀

Comment (kazo0) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: Hit that approve button :)

**Review by MartinZikmund — APPROVED** [review]
🚀🚀🚀

File: src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs (hunk 2 above)
Comment (Xiaoy312) [reply src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/App.xaml.cs:59]: > ... And, be sure to subscribe to the repository and leave a comment below. XD

Jokes asides, I checked github and azure devops and there seems no open issue/work-item regarding this.
